index.html

```diff
@@ -50,39 +52,96 @@ <h3>Furahia Insurance <br><span class="logo-writing2">Since 1965</span></h3>
       <h2 id="intro1">Enjoy life na Furahia Insurance!!</h2>
     </div>
   </header>
-  <section>
-    <article>    
-      <h1 id="intro2">PROMOTING A LIFE <br> FREE LIFESTYLE <br> 2022</h1>  
-      <p>Peace of mind comes from knowing you and your family are covered and protected by the best health insurance cover</p>
+  <main>
+    <article>
+      <h1 id="intro2">PROMOTING A LIFE <br> FREE LIFESTYLE <br> 2022</h1>
+      <p class="intro-words">Peace of mind comes from knowing you and your family are covered and protected by the best
+        health insurance cover</p>
       <h1 id="intro3">50 YEARS IN BUSINESS </h1>
       <P id="intro4">We advise, Insure and Invest</P>
     </article>
-    <section class="row products-categories">
+    <section class="products-categories">
       <h3 class="products1">Main Products</h3>
 
-      <div class="product-life">
-        <img class="product-img" src="./imgs/life.jpeg" alt="Life Insurance">
-        <h2 class="product-name">Life</h2>
-        <p class="product-description">Our job is to help you choose the right insurance cover for you so that you may Live Free!</p>
+      <div class="insurance-products">
```

Comment:
- [ ] Great job adding the cards as requested by the previous reviewer, however, the cards are of not the same size, kindly fix this issue.

Your design
![image](https://user-images.githubusercontent.com/81932478/169654817-55a3725a-0147-40c9-9185-b039fb649109.png)


Provided design
![image](https://user-images.githubusercontent.com/81932478/169654867-612c86f9-198c-431a-abec-38318c749676.png)

## *Note: the last card is not the same size

Reply:
@Munsa1 , This is now fixed